dev-python/dbuspython/dbuspython-1.2.14.recipe

```diff
@@ -0,0 +1,110 @@
+SUMMARY="The original dbus implementation for Python"
+DESCRIPTION="dbus-python is the original Python binding \
+for dbus, the reference implementation of the D-Bus protocol."
+HOMEPAGE="https://www.freedesktop.org/wiki/Software/DBusBindings/#python"
+COPYRIGHT="2003-2006 Red Hat Inc.
+           2003 David Zeuthen
+           2004 Rob Taylor
+           2004 Anders Carlsson
+           2005 Colin Walters
+           2005-2019 Collabora Ltd.
+           2016 Simon McVittie"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://dbus.freedesktop.org/releases/dbus-python/dbus-python-$portVersion.tar.gz"
+CHECKSUM_SHA256="b10206ba3dd641e4e46411ab91471c88e0eec1749860e4285193ee68df84ac31"
+SOURCE_DIR="dbus-python-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	dbuspython = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libdbus_1
+	lib:libglib_2.0
```

Comment:
These should be moved to the `_python*` sub-packages, as the main package contains no libraries, only the sub-packages do.